examples/shallow_2d/plot.py

```diff
@@ -0,0 +1,80 @@
+r"""Convenience routines for easily plotting with VisClaw."""
```

Comment:
Is this meant to be in this PR?

Reply:
How do I make these disappear in my next PR? I squashed most of my unnecessary commits into one or two now, but I am afraid these will still appear when I push my repo up to my github